src/AppError.tsx

```diff
@@ -3,17 +3,24 @@ import { Route, Switch } from 'react-router-dom';
 import { APP_PATHS } from 'src/constants';
 import useQuery from './utils/useQuery';
 import ErrorContent from './ErrorContent';
+import { Provider } from 'react-redux';
+import AppBootstrap from './AppBootstrap';
+import { store } from './redux/store';
 
 export default function AppError() {
   return (
-    <Switch>
-      <Route exact path={APP_PATHS.ERROR_FAILED_TO_FETCH_ORG_DATA}>
-        <ErrorContent />
-      </Route>
-      <Route path={APP_PATHS.ERROR}>
-        <QueryParamsError />
-      </Route>
-    </Switch>
+    <AppBootstrap>
+      <Provider store={store}>
```

Comment:
what is the store for? app version detector?

Reply:
I don't think we are using the store here, just wanted to have same initialization as App, but probably we can remove it

Reply:
no worries, was just curious.. we can leave it if we want same initialization / consistency